mozilla-custom/ga-snippet.js

```diff
@@ -69,5 +69,22 @@ Mozilla.dntEnabled = function(dnt, ua) {
         ga('create', 'UA-36116321-4', 'auto');
         ga('set', 'dimension1', blogName);
         ga('send', 'pageview');
+
+        // Activate new GA4 measurement ID through gtag script
+        var measurementId = 'G-X4N05QV93S';
+
+        (function(doc, tag, url) {
+            var newScript = doc.createElement(tag);
+            newScript.async = 1;
+            newScript.src = url;
+            existingScript = doc.getElementsByTagName(tag)[0]
```

Comment:
Whoops. This needed an extra fix.

```suggestion
            var existingScript = doc.getElementsByTagName(tag)[0]
```